src/AWS.Deploy.CLI/ServerMode/Services/SessionOrchestratorInteractiveService.cs

```diff
@@ -37,5 +41,15 @@ public void LogMessageLine(string message)
         {
             _hubContext.Clients.Group(_sessionId).OnLogMessageLine(message);
         }
+
+        public void ErrorDataReceived(ProcessStartInfo processStartInfo, string data)
+        {
+            LogMessageLine(data);
```

Comment:
LogErrorMessageLine instead of LogMessageLine?

Reply:
https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.CLI/Utilities/CommandLineWrapper.cs#L88

we don't use `LogErrorMessageLine` in current version of `CommandLineWrapper`. Intention is to not change any behavior in refactor PR.